api/system-openssh/hints

```diff
@@ -20,11 +20,31 @@
 # along with NethServer.  If not, see COPYING.
 #
 
-. /usr/libexec/nethserver/api/lib/helper_functions
+require '/usr/libexec/nethserver/api/lib/helper_functions.pl';
```

Comment:
We have a Perl module since a long time...

```suggestion
use NethServer::ApiTools qw(hints success error readInput safe_decode_json);
```

Reply:
done